### DISCUSSION TITLE
Does it support transcribing .flv types of files into text?

### DISCUSSION THREAD

**Discussion by drapeol** [openai/whisper#2101] [Show and tell]
Title

**Comment by glangford** [comment]
whisper uses `ffmpeg` to process audio. If ffmpeg can decode it, then whisper should be able to transcribe it. The question is what audio codec is being used in the .flv files.

You can see what formats your version of ffmpeg supports using
`ffmpeg -formats`

See also

FFMPEG supported media types
http://trac.ffmpeg.org/wiki/SupportedMediaTypesInFormats

StackOverflow - Which are the FLV supported audio types?
https://stackoverflow.com/questions/4716203/which-are-the-flv-supported-audio-types

**Reply by drapeol** [reply]
Thank you for your reply. I don't know the ffmpeg  but you have pointed me in a direction to learn it. I will try to learn and use it afterward.

**Comment by a2nath** [comment]
Does whisper have a built-in noise rejection/background rejection algo? I'm thinking of adding this feature in my automation script by using the ffmpeg separately and its built-in NR algo if whisper does not support it.

**Comment by glangford** [comment]
@a2nath No there is no noise rejection used currently. Here is the ffmpeg command used to load audio:

https://github.com/openai/whisper/blob/ba3f3cd54b0e5b8ce1ab3de13e32122d0d5f98ab/whisper/audio.py#L45
